frontend/src/lib/components/Support/SupportForm.tsx

```diff
@@ -7,7 +7,7 @@ import {
     Link,
 } from '@posthog/lemon-ui'
 import { useActions, useValues } from 'kea'
-import { Form } from 'kea-forms'
+import { Field, Form } from 'kea-forms'
```

Comment:
in which case (conversation below) this is the wrong import, no?

this should be `LemonField` still

or I've misunderstood 🤣